Turf simplify hangs on this input.

Running v4.6.0, this input caused simplify to hang the nodejs process in a compute loop:
```
# coffeescript
ts=require '@turf/simplify'
ts {"type":"Feature","geometry":{"coordinates":[[[[0,90],[0,90],[0,90],[0,90],[0,90],[0,90],[0,90],[0,90],[0,90],[0,90],[0,90]]]],"type":"MultiPolygon"},"properties":{}},1
```
(I realize it's unusual input.)

@jfgorski I've tried to create MultiPolygon's using your coordinates and they are all "invalid", so unless you can provide valid geometries then this wouldn't be fixed.

```js
multiPolygon([[[[0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90]]]])
multiPolygon([[[[0, 1], [0, 2], [0, 3], [0, 2.5], [0, 1]]]])
multiPolygon([[[[0, 1], [0, 1], [1, 2], [0, 1]]]])
```
All of them throws this error:

```
/Users/mac/Github/turf/packages/turf-simplify/index.js:132
            throw new Error('invalid polygon');
            ^

Error: invalid polygon
    at /Users/mac/Github/turf/packages/turf-simplify/index.js:132:19
    at Array.map (native)
    at simplifyPolygon (/Users/mac/Github/turf/packages/turf-simplify/index.js:127:24)
    at /Users/mac/Github/turf/packages/turf-simplify/index.js:92:20
    at Array.map (native)
    at simplify (/Users/mac/Github/turf/packages/turf-simplify/index.js:91:47)
    at /Users/mac/Github/turf/packages/turf-simplify/index.js:54:9
    at geomEach (/Users/mac/Github/turf/packages/turf-meta/index.js:460:17)
    at module.exports (/Users/mac/Github/turf/packages/turf-simplify/index.js:53:5)
    at Test.t (/Users/mac/Github/turf/packages/turf-simplify/test.js:80:14)
npm ERR! Test failed.  See above for more details.
```

@jfgorski Please don't hesitate to re-open this issue if you can replicate these errors with valid geometries using `@turf/helpers`.

--- a/packages/turf-simplify/test.js
+++ b/packages/turf-simplify/test.js
@@ -75,8 +75,8 @@ test('simplify -- issue #555', t => {
 
 test('simplify -- issue #918', t => {
     // simplify hangs on this input #918
-    t.skip('issue #918', multiPolygon([[[[0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90]]]]));
-    t.skip('issue #918', multiPolygon([[[[0, 1], [0, 2], [0, 3], [0, 2.5], [0, 1]]]]));
-    t.skip('issue #918', multiPolygon([[[[0, 1], [0, 1], [1, 2], [0, 1]]]]));
+    t.throws(() => simplify(multiPolygon([[[[0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90], [0, 90]]]])), /invalid polygon/, 'invalid polygon');
+    t.throws(() => simplify(multiPolygon([[[[0, 1], [0, 2], [0, 3], [0, 2.5], [0, 1]]]])), /invalid polygon/, 'invalid polygon');
+    t.throws(() => simplify(multiPolygon([[[[0, 1], [0, 1], [1, 2], [0, 1]]]])), /invalid polygon/, 'invalid polygon');
     t.end();
 });

